client/my-sites/plans-comparison/plans-comparison-col-header.tsx

```diff
@@ -77,13 +87,43 @@ export const PlansComparisonColHeader: React.FunctionComponent< Props > = ( {
 	originalPrice,
 	children,
 	translate,
+	isExperiment,
 } ) => {
 	const isDiscounted = typeof originalPrice === 'number';
 
 	return (
 		<th scope="col">
 			<PlanTitle>{ plan.getTitle() }</PlanTitle>
-			<PlanDescription>{ plan.getDescription() }</PlanDescription>
+
+			<PlanDescription>
```

Comment:
`PlanDescription` is a [styled paragraph tag](https://github.com/Automattic/wp-calypso/pull/64239/files#diff-b6b238d910a038a06ac8f6dedcccf8ba75991602a186b797fbf9d59d771fb9ceR38), so this technically nests a paragraph inside a paragraph, which is against HTML spec. You can update `const PlanDescription = styled.p` to `const PlanDescription = styled.div` to get around this for the test, although that might require some style adjustment.